junction/proposals/admin.py

```diff
@@ -32,26 +54,27 @@ def get_queryset(self, request):
         return qs.filter(
             conference_reviewer__conference__in=[m.conference for m in moderators]
         )
-    
+
     def get_form(self, request, obj=None, **kwargs):
         form = super(ProposalSectionReviewerAdmin, self).get_form(request, obj, **kwargs)
-        form.base_fields['conference_reviewer'].queryset = ConferenceProposalReviewer.objects.all().order_by('-created_at')
+        form.base_fields['conference_reviewer'].queryset = ConferenceProposalReviewer.objects.all().order_by(
+            '-created_at')
         form.base_fields['proposal_section'].queryset = ProposalSection.objects.all().order_by('-created_at')
         return form
 
 
 @admin.register(models.ProposalType)
 class ProposalTypeAdmin(AuditAdmin):
     list_display = (
-        "name",
-        "active",
-        "start_date",
-        "end_date",
-    ) + AuditAdmin.list_display
+                       "name",
```

Comment:
Can you please remove these additional white spaces ? There are some more in the subsequent lines.

Reply:
Removed.